docs/INSTALL.md

```diff
@@ -0,0 +1,658 @@
+      1. [Proper SSL configuration](#proper-ssl-configuration)
+   1. [Serve security related Headers](#serve-security-related-headers)
+   1. [Report security issues](#report-security-issues)
+1. [Customizing ILIAS](#customizing-ilias)
+   1. [Plugin Repository](#plugin-repository)
+1. [Upgrading ILIAS](#upgrading-ilias)
+   1. [Minor Upgrade](#minor-upgrade)
+   1. [Major Upgrade](#major-upgrade)
+   1. [Information on Updates](#information-on-updates)
+1. [Upgrading Dependencies](#upgrading-dependencies)
+   1. [PHP](#php)
+   1. [MySQL](#mysql)
+   1. [ImageMagick](#imagemagick)
+1. [Contribute](#contribute)
+   1. [Pull Requests](#pull-requests)
+   1. [Reference System](#reference-system)
+
+<!-- /MarkdownTOC -->
+
+<a name="system-requirements"></a>
+# System Requirements
+
+The necessary hardware to run an ILIAS installation is always dependent from the number of users and the kind of usage.
+
+<a name="cpu"></a>
+## CPU
+
+We RECOMMEND a common dual core server CPU.
+
+<a name="memory"></a>
+## Memory
+
+Memory requirements are greatly variable, depending on the number of users and server activity. We RECOMMEND a minimum of 4096 MB.
+
+<a name="harddrive"></a>
+## Harddrive
+
+We RECOMMEND 250 GB for usual ILIAS deployments. 25 GB would be used by the operating system and ILIAS itself. 225 GB would remain for the database and files.
+
+<a name="bandwidth"></a>
+## Bandwidth
+
+We RECOMMEND at least 100 Mbit/sec. for the web server WAN connection.
+
+<a name="recommended-setup-for-running-ilias"></a>
+## Recommended Setup for Running ILIAS
+
+For best results we RECOMMEND:
+
+  * Debian GNU Linux 8 / Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
+  * MySQL 5.5 / MariaDB
+  * PHP 5.6+
+  * Apache 2.2+ with mod_php
+  * ImageMagick 6.x+
+  * php5-gd, php5-xsl, php5-mysql
+  * OpenJDK 7+
+  * zip, unzip
+  * git
+
+<a name="supported-platforms"></a>
+## Supported Platforms
+
+Please note that different configurations SHOULD be possible, but it might be harder to find someone who can help when things go south. You SHALL NOT use a different configuration unless you are an experienced system administrator.
+
+<a name="server"></a>
+### Server
+
+  * Server OS: Linux
+  * Web Server: Apache 2 (mod_php, php-fpm)
+  * Databases: MySQL/MariaDB 5.0+ and Galera (experimental), Oracle 10g+ (experimental), PostgreSQL (experimental)
+  * PHP: Version 5.5+ and 7.0+ are supported
+  
+See http://www.ilias.de/docu/goto.php?target=lm_367&client_id=docu for details.
+  
+<a name="client"></a>
+### Client
+
+  * Desktop: Windows 7+, MacOS X 10.7+, Linux
+  * Web Browser: IE11+, Microsoft Edge, Firefox 14+, Chrome 18+, Safari 7+
+
+<a name="database-recommendations"></a>
+## Database Recommendations
+
+We RECOMMEND to use MySQL/MariaDB with the following settings:
+
+  * InnoDB storage engine
+  * utf8_general_ci
+  * STRICT_TRANS_TABLES or STRICT_ALL_TABLES disabled (on MySQL 5.6.x)
```

Comment:
Not sure if a RECOMMEND is enough for the current STRICT_..._TABLES-Situations since a lot of things break without this.